### ISSUE TITLE
Full CPU load and slow processing performance of data from websocket if using long-lived connection via dispatcher=rel option

### ISSUE THREAD

**Issue by yaroshoris** [websocket-client/websocket-client#985]
Hi, I faced with strange behaviour, trying to set up long-lived connection (as it's written in docs).

My OS is Ubuntu 24 and Python 3.12.3
websocket-client==1.8.0
rel==0.4.9.17

There is the minimal code for reproducing this bug. I have tried to reproduce on Windows 11 and got the same behavior.
```
import datetime
import json
import websocket
import rel

rel.initialize(["select"])
rel.initialize(options=["verbose"])

class Binance():
    def __init__(self):

        self.wsapp = websocket.WebSocketApp(
            url="wss://fstream.binance.com/stream?streams=btcusdt@bookTicker/ethusdt@bookTicker/bnbusdt@bookTicker",
            on_message=self.on_message,
            on_open=self.on_open,
            on_close=self.on_close,
            on_error=self.on_error,
            on_ping=self.on_ping,
            on_pong=self.on_pong)


    def on_open(self, wsapp):
        print("Websocket connection open")


    def on_message(self, wsapp, message):

        data = json.loads(message)
        event_time = data["data"]["E"]
        now = int(datetime.datetime.now().timestamp() * 1000)
        lag = now - event_time

        print(f"{datetime.datetime.now()} --- Current lag is {lag}")

    def on_error(self, wsapp, error):
        print(f"Error in websocket: {error}")

    def on_close(self, wsapp, close_status_code, close_msg):
        print(f"Websocket connection close {close_status_code} and msg {close_msg}")

    def on_ping(self, wsapp, message):
        print("received ping from server")

    def on_pong(self, wsapp, message):
        print("received pong from server")

    def run_wsapp(self):
        websocket.enableTrace(False)
        self.wsapp.run_forever(dispatcher=rel, reconnect=5) 
        rel.signal(2, rel.abort) 
        rel.dispatch()


if __name__ == "__main__":
    client = Binance()
    client.run_wsapp()

```

In Ubuntu, using the command  **netstat -npt | grep python** I can see that buffer grows enermously quick. 
Also, the script that I provide for test print LAG from current TS and TS that is sended by exchange in message.

If I use the script without specifing dispatcher  self.wsapp.run_forever(reconnect=5), all work as expected.

Thanks.

**Comment by bubbleboy14** [comment]
@yaroshoris ah, good point. This issue is fixed in rel 0.4.9.18 - give it a try and LMK how it goes!

**Comment by yaroshoris** [comment]
@bubbleboy14 thanks, now the issue is resolved. All are working as expected. Thanks and much appreciate your involving:)
